projects/netgrif-components-core/src/lib/header/task-header/task-header.service.ts

```diff
@@ -19,8 +20,9 @@ export class TaskHeaderService extends AbstractHeaderService implements OnDestro
                 preferences: UserPreferenceService,
                 logger: LoggerService,
                 @Optional() viewIdService: ViewIdService,
+                @Optional() protected overflowService: OverflowService,
```

Comment:
why is this `protected` if it is passed to `super` ?

Reply:
Done